products/mozilla_org/test_mozilla_org.md

```diff
@@ -36,32 +36,86 @@ It’s highly advised you to ask other community members to conduct peer review
 
 You can make linguistic changes directly in [Pontoon](https://pontoon.mozilla.org/projects/mozillaorg/), [Locamotion](https://mozilla.locamotion.org/projects/mozilla_lang/), or [GitHub](https://github.com/mozilla-l10n/www.mozilla.org/).
 
-### Functionality Testing
+### Functionality testing
 * Click the download link, you should be able to download the product in your language, if it is localized, such as Firefox
 * Font support and readability
 * Footer: verify that the translation of the link is coherent and the link is functional
-* Language list: Is your language listed as one of the options? Check https://www-dev.allizom.org/en-US/ to confirm
+* Language list: Is your language listed as one of the options? Check https://www-dev.allizom.org/ to confirm
```

Comment:
Commenting again here, since doing it on a commit is confusing: this one actually needs to point to /en-US